test/groovy/DebugReportArchiveTest.groovy

```diff
@@ -0,0 +1,53 @@
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import util.*
```

Comment:
I think we do not use star imports

Reply:
No idea how that ended up there... will fix.